modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/related/models/test/ObjectRelatedModelsProviderTest.java

```diff
@@ -437,6 +437,19 @@ public void testObjectEntry1toMObjectRelatedModelsProviderImpl()
 			_objectRelationship.getObjectRelationshipId(),
 			objectEntry8.getObjectEntryId());
 
+		// Empty 1:M relationship entry
```

Comment:
@G-Sa Brian suggested this new comment? Because for me, it still doesn't follow the same pattern as the other comments in this class.

Maybe we should consider moving this test to another class, like `ObjectEntryLocalServiceTest`.

In addition, based on the [commit](https://github.com/brianchandotcom/liferay-portal/pull/128980/commits/f35ee36a89823b18c22979e68369f43fc2c78451) fixing the related LPP,  we should write a playwright test instead. What do you think?

Reply:
We can try to keep both, but I would prioritize the playwright.